.github/workflows/kind.yaml

```diff
@@ -17,12 +17,34 @@ jobs:
         run: |
           export GO111MODULE=on
           export PATH=$PATH:$(go env GOPATH)/bin
+          # Print cluster info
           kubectl cluster-info
           kubectl get pods -n kube-system
           kubectl wait --for=condition=Ready pods --all -n kube-system
-          make generate
-          make install
-          timeout 1m make run || retVal=$?
+
+          # TODO: Move to Makefile - Cert manager
+          kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.7.0/cert-manager.yaml
+          kubectl wait --for=condition=Ready pods --all -n cert-manager
+
+          # Deploy flotta operator
+          make build
+          IMG=flotta-operator:latest make docker-build
+          kind load docker-image flotta-operator:latest
+          timeout 1m make deploy IMG=flotta-operator || retVal=$?
           if [ $retVal -ne 124 ]; then
             echo "Make run failed"; exit 1
           fi
+          kubectl wait --timeout=120s --for=condition=Ready pods --all -n flotta
+          kubectl port-forward deploy/flotta-operator-controller-manager -n flotta --address 0.0.0.0 8888:8888 &
```

Comment:
for minikube we enable the nginx ingress controller and ingress CR to expose the service.
we'd might need a different [way to install](https://kind.sigs.k8s.io/docs/user/ingress/#ingress-nginx) for kind 
```
kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
```
unless we prefer port-forwarding instead for testing.

Reply:
I will add support for kind ingress. The port-forwarding was just quick working solution.